CONTRIBUTING.md

```diff
@@ -48,20 +61,29 @@ Improvements to existing functionality are tracked as [GitHub issues using the U
 
 ### Set up your dev environment
 
-Your dev environment is set up by [tox](https://tox.wiki/en/latest/), an environment orchestrator which allows for setting up environments for and invoking builds, unit tests, formatting, linting, etc. Install tox with:
+The following tools are required:
+
+- [git](https://git-scm.com)
+- [python](https://www.python.org) (v3.9+)
```

Comment:
Q - we do also still support 3.8. is there a particular reason 3.9 is advised?

Reply:
Good catch. Updated.